examples/echoserver/echoserver.c

```diff
@@ -188,9 +208,169 @@ typedef struct {
 #endif
     byte channelBuffer[EXAMPLE_BUFFER_SZ];
     char statsBuffer[EXAMPLE_BUFFER_SZ];
+#ifdef WOLFSSH_WOLFSENTRY_HOOKS
+    struct wolfsentry_data* pdata;
+#endif
 } thread_ctx_t;
 
 
+#ifdef WOLFSSH_WOLFSENTRY_HOOKS
+
+#define WOLFSENTRY_INVALID_PASSWORD     -1
+#define WOLFSENTRY_VALID_PASSWORD       1
+
+/**
+ *  Submit an authentificaion result event into wolfsentry and
```

Comment:
Spelling - "authentificaion" should be "authentication"

Reply:
thanks. fixed